### ISSUE TITLE
Unmanaged/Ingest bug in vmware context

### ISSUE THREAD

**Issue by DK101010** [apache/cloudstack#4539]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
api,core,vmware
~~~

##### CLOUDSTACK VERSION

~~~
version > 4.13
~~~

##### OS / ENVIRONMENT

vmware

##### SUMMARY

During the unmanagement process of an vmware vm, I'll get an exception with Unknown Error. But unmanaged process itself works correctly, after this VM is successfully unmanaged. 

Also this VM's haven't a vm_destroy event in the database and in the usage context no end date.

##### STEPS TO REPRODUCE

~~~
Use a vmware environment, create a VM in cloudstack with a custom offering and unmanage this vm.
~~~

##### EXPECTED RESULTS

~~~
When I unmanage a vmware vm in cs, process ends without exceptions and get a destroy event for correct usage informations.
~~~

##### ACTUAL RESULTS

~~~
exception during the unmanaged process, although process is successfully. Usage data for unmanaged vm are correctly
~~~

**Comment by DaanHoogland** [comment]
@shwstppr can you have a look ?

**Comment by rhtyd** [comment]
@shwstppr can you review the critical-ity of this?

**Comment by shwstppr** [comment]
@DaanHoogland @rhtyd this bug can only be in master as Unmanaging feature was added after 4.14 https://github.com/apache/cloudstack/pull/4103.
@DK101010 can you share some logs, your environment details?

**Comment by DK101010** [comment]
I have created a pull request for this

**Comment by DK101010** [comment]
here is the pr https://github.com/apache/cloudstack/pull/4540

**Comment by rhtyd** [comment]
The PR is merged https://github.com/apache/cloudstack/pull/4540 should we close this @DK101010 ?

**Comment by DK101010** [comment]
> The PR is merged #4540 should we close this @DK101010 ?

Hi Rohit, yes, you can close it.
